docs/build.sh

```diff
@@ -0,0 +1,4 @@
+pip install--user sphinx
```

Comment:
```diff
- pip install--user sphinx
+ pip install --user sphinx

Reply:
We could leverage the state-of-art best theme now?

Install Furo in the documentation's build environment.

```bash
pip install install --user sphinx furo
```

Then update the html_theme in conf.py.

```python
html_theme = "furo"
```

ref: https://github.com/pradyunsg/furo

furo theme will look like https://devguide.python.org/

Reply:
I can prepare for some guides later with this as the base! And afterwards, we could at some point publish this to readthedocs.io